pkg/config/config.go

```diff
@@ -1231,7 +1231,9 @@ func GetMultipleEndpoints() (map[string][]string, error) {
 }
 
 func bindEnvAndSetLogsConfigKeys(config Config, prefix string) {
-	config.BindEnv(prefix + "logs_dd_url") // Send the logs to a proxy. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
+	config.BindEnv(prefix + "logs_dd_url")         // Send the logs to a proxy. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
+	config.BindEnv(prefix + "logs_backup_dd_url")  // Send the logs to an alternative intake. Must respect format '<HOST>:<PORT>' and '<PORT>' to be an integer
```

Comment:
This line is over 150 chars now - probably should have the comment aligned with the code